content/participants/ttjc@gauravmishra-17/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Gaurav Mishra
+type: "participant"
+name: "Gaurav Mishra"
+portfolio: ""
+github-link: "https://github.com/gauravmishra-17"
+linkedin-link: "https://www.linkedin.com/in/gauravmishra17/"
+looking-for: "job"
+description: "Competitive Coder | Flutter "
+---
+
+Hi! I have been doing Competitive coding since past few months. Currently I belong to the unfortunate class of 2020, with my joining delayed, and possible revocation, who knows. I started my development journey by starting flutter. But in this month of June, I aim to complete this challenge by making projects on various tech. And I aim to become a FullStack Developer by the end of this month. Anyhow.
+
+### Introduction
+
+I am from Lucknow, studying in Ghaziabad. My current goal is to win a job offer from a good product based company.
+
+## Why are you taking this challenge?
+
+I want to become a good developer and secure a better job offer than present by the end of june.
+
+## Something interesting about you?
+
+I can be too much practical person, where everyone things I am rude.
+
+### Help I need?
+
+I just want someone to help me understand the flowchart of building a project. I get lost to easily.
+
+### How to contact me?
+
+- Twitter
```

Comment:
Hey! @gauravmishra-17 please add your twitter link here and if you don't have one then remove it.

Reply:
@khusharth I have added the twitter link

Reply:
Hello maintainers!? will my pr get merged? Please accept my pr. I edited my line as it was suggested to be done before the deadline.

Reply:
@khusharth  @gkanishk

Reply:
Chill man!!
Reviewing PR takes time, should have patience.

Reply:
man I was freaked about the deadline.